crates/bevy_feathers/src/font_styles.rs

```diff
@@ -0,0 +1,62 @@
+//! A framework for inheritable font styles.
+use bevy_app::Propagate;
+use bevy_asset::{AssetServer, Handle};
+use bevy_ecs::{
+    component::Component,
+    lifecycle::Insert,
+    observer::On,
+    system::{Commands, Query, Res},
+};
+use bevy_text::{Font, TextFont};
+
+use crate::handle_or_path::HandleOrPath;
+
+/// A component which, when inserted on an entity, will load the given font and propagate it
```

Comment:
This makes sense given the current behavior (both lack of font inheritance and wanting to express an AssetPath instead of needing to manually load a handle on an asset server, which BSN will fix). I think we might want to consider making "text style" inheritance happen implicitly by default, but thats not something to resolve here.

Reply:
Sure, let's leave this for a follow-up.